starter-analytics-postgres/src/test/groovy/io/micronaut/starter/analytics/postgres/StoreGeneratedProjectStatsSpec.groovy

```diff
@@ -92,8 +94,8 @@ class StoreGeneratedProjectStatsSpec extends Specification implements TestProper
         wrongApiKeyClient.applicationGenerated(generated).get()
 
         then:
-        Exception e = thrown()
-        e.cause.message == "Unauthorized"
+        ExecutionException e = thrown()
```

Comment:
Can you not use?

```suggestion
        HttpClientResponseException e = thrown()
```

Reply:
[Same here](https://github.com/micronaut-projects/micronaut-starter/pull/1980/files/c43ea85f87b37a1ef9377e450cce1f1e5182d2a9..a33d3c7099091b53317e74de4ff5e6b25d233f72#r1275001093).